k256/src/arithmetic/projective.rs

```diff
@@ -220,6 +257,7 @@ impl ProjectivePoint {
         ret
     }
 
+    #[cfg(not(all(target_os = "zkvm", target_arch = "riscv32")))]
     /// Doubles this point.
     #[inline]
```

Comment:
```suggestion
    /// Doubles this point.
    #[cfg(not(all(target_os = "zkvm", target_arch = "riscv32")))]
    #[inline]
```

Reply:
I only marked one of this pattern, but I saw a few more

Reply:
got them in commits below :)